src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.html

```diff
@@ -0,0 +1,97 @@
+<div class="research">
+  <div class="search-imput"><input type="text" placeholder="Search" (keyup)="applyFilter($event.target.value)" /></div>
+  <div class="icon-style"><i class="fa fa-search" aria-hidden="true"></i></div>
+</div>
+<div class="btn-group">
+  <button type="button" class="btn btn-default" class="amount-items"><i class="fa fa-angle-down" aria-hidden="true"></i> 10</button>
+  <button type="button" class="btn btn-default" class="filter-button">
+    <div><mat-icon>filter_list</mat-icon><span>Фільтрувати</span></div>
+  </button>
+  <button type="button" class="btn btn-default" class="view-table-btn" data-toggle="dropdown">
+    <mat-icon>create</mat-icon>Вид таблиці
+  </button>
+  <ul class="dropdown-menu">
+    <li>Інформація про замовлення</li>
```

Comment:
Move all text to separate file in the asset folder.

Reply:
skip